www/index.html

```diff
@@ -49,6 +49,11 @@ <h1 class="mt-5">Firefox vs. Chrome</h1>
               <td><a href="https://twitter.com/PowerDNS_Bert/status/1170638582471675906" target="_blank">Yes!</a></td>
               <td>No.</td>
             </tr>
+            <tr>
+              <th scope="row">Is incompatible with privacy-saving extensions like uBlock Origin</th>
```

Comment:
```suggestion
              <th scope="row">Prevents anti-tracking and anti-ad extensions like uBlock Origin from working fully</th>
```

ust two wording changes:
* add "fully", because in Chome/ium limited support may still be there
* "privacy-saving" is not proper English AFAIK
* also consider to maybe remove the example